tests/conftest.py

```diff
@@ -174,13 +174,14 @@ def tmpconfdir(
     )
     assert path.is_file(), 'WTH.. `brokers.toml` not created!?'
 
-    return tmpconfdir
+    yield tmpconfdir
 
     # NOTE: the `tmp_dir` fixture will wipe any files older then 3 test
     # sessions by default:
     # https://docs.pytest.org/en/6.2.x/tmpdir.html#the-default-base-temporary-directory
     # BUT, if we wanted to always wipe conf dir and all contained files,
-    # rmtree(str(tmp_path))
+    from shutil import rmtree
```

Comment:
Hmm, my one question was whether we ended up needing this?

I can't remember if we are auto-injecting the per-test user config tmpdir or it stays pseudo-global in between each test scope.

Not a huge deal for now (since explicit cleanup is normally always better).

Reply:
I'm not sure why we added this. but I commented it and tested, it still working there is no error. maybe we don't need it

Reply:
Yeah, so i think that means the tests are isolated in terms of the user-dir created per test. So yeah, leave it commented as it was for now 👍🏼